config/gradle/jre.gradle

```
@@ -2,8 +2,8 @@
 // SPDX-License-Identifier: Apache-2.0
 
 // Uses Bellsoft Liberica JRE
-// https://bell-sw.com/pages/liberica_install_guide-11.0.7/
-def jdkVersion = '11.0.12+7' // jre version haven't jmod (needed for jlink/jpackager)
+// https://bell-sw.com/pages/liberica_install_guide-11.0.15.1/
+def jdkVersion = '11.0.15.1+2'
```

Looks like we have [11.0.16.1](https://bell-sw.com/pages/liberica-release-notes-11.0.16.1/) already... 

```suggestion
// https://bell-sw.com/pages/liberica_install_guide-11.0.16.1/
def jdkVersion = '11.0.16.1+1'
```